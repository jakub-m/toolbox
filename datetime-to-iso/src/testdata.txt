foo=datetime.datetime(2025, 5, 20, 8, 16, 5, 743193, tzinfo=datetime.timezone.utc) bar=bar

