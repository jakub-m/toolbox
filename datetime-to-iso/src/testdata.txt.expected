single datetime: foo=2025-05-20T08:16:05.743Z bar=bar
two datetimes: foo=2025-05-20T08:16:05.743Z 2025-05-20T08:16:05.743Z bar=bar
no seconds: foo=2025-05-20T08:16Z bar=bar

